rotkehlchen/inquirer.py

```diff
@@ -818,14 +832,20 @@ def find_curve_pool_price(
 
         May raise:
         - RemoteError
+        - AssertionError if used for a token whose chain_id is not in CURVE_CHAIN_IDS
         """
-        ethereum = self.get_evm_manager(chain_id=lp_token.chain_id)
-        ethereum.assure_curve_cache_is_queried_and_decoder_updated()  # type:ignore  # ethereum is an EthereumManager here
+        assert lp_token.chain_id in CURVE_CHAIN_IDS, f'{lp_token} is not on a curve support chain'
```

Comment:
you did not fix the typo here
```suggestion
        assert lp_token.chain_id in CURVE_CHAIN_IDS, f'{lp_token} is not on a curve supported chain'
```

Reply:
oh I did this and then had to reset everything because many things were messed while I tried to fix the typing, and then forgot to do this again